seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -856,7 +856,7 @@
       "id": "bad87fee1348bd9aede08817",
       "title": "Nest an Anchor Element within a Paragraph",
       "description": [
-        "Again, here's a diagram of an <code>a</code> element for your reference:",
+        "Below is the diagram showing that you can nest an <code>a</code> element inside a <code>p</code> element:",
```

Comment:
Again, instead of using diagrams (images) we'd like to explain all of this just using text. Please see if you can obviate the need for an image through clear description.